### PR TITLE
Add handing of onnx metadata

### PR DESCRIPTION
**Pull request by sovrasov**
# What does this PR do?

This PR slightly modifies python OV adapter to read an analog of rt_info from onnx.
Technical details:
- rt_info is stored in onnx_model.metadata_props
- key of the metadata property is a string containing space-separated xml tags from the original rt_info path, for instance, `model_info confidence_threshold`, `model_info hierarchical_config`. All values of attrs are stored as strings, as in rt_info

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by sovrasov** [comment]
Implementation is finished, now waiting for the model to be uploaded to s3.

File: model_api/python/openvino/model_api/adapters/openvino_adapter.py [hunk 1]
```diff
@@ -18,13 +18,16 @@
 from pathlib import Path
 from typing import Dict, Set, Tuple
 
+import onnx
```
Comment (Wovchena) [inline model_api/python/openvino/model_api/adapters/openvino_adapter.py:21]: Please, move the import under `if Path(self.model_path).suffix == ".onnx" and not weights_path:` to make it optional

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: model_api/python/openvino/model_api/adapters/openvino_adapter.py (hunk 1 above)
Comment (sovrasov) [reply model_api/python/openvino/model_api/adapters/openvino_adapter.py]: Done

**Review by Wovchena — APPROVED** [review]

**Comment by sovrasov** [comment]
@Wovchena tests are passing now

**Pull request merged**